.travis.yml

```diff
@@ -1,9 +1,8 @@
 sudo: false
 language: node_js
 node_js:
-  - "iojs"
-  - "0.10"
-  - "0.12"
```

Comment:
aren't these old versions installed on some systems by default? I thought they were on mac os x but I can't find any evidence of that so maybe not.